vs-build/FAST-farm/FAST-Farm.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
+# Visual Studio Version 16
```

Comment:
Do we want to upgrade the visual studio version?

Reply:
I debated upgrading this. I've been holding everything back on older versions for better coverage of our user base. But honestly, I don't know who would be on VS 2017 or older at this point.

Version 16 corresponds to VS 2019.

Reply:
I'm good with updating. I just wanted to check that this was intentional, I've done it accidentally before.

Reply:
> I debated upgrading this. I've been holding everything back on older versions for better coverage of our user base. But honestly, I don't know who would be on VS 2017 or older at this point.
> 
> Version 16 corresponds to VS 2019.
LOL, I'm using VS 2017.

Reply:
@bjonkman, I'll roll the VS version back.  There really aren't any substantial changes between solution files for versions other than this one number.

Reply:
It really doesn't matter. The 2017 version is very old. :-)